cypress/e2e/hub/collections-detail.cy.ts

```diff
@@ -29,141 +29,84 @@ describe('Collections Details', () => {
     cy.deleteHubNamespace({ ...namespace, failOnStatusCode: false });
   });
 
-  beforeEach(() => {
-    collectionName = randomE2Ename();
-    cy.navigateTo('hub', Collections.url);
-    cy.verifyPageTitle(Collections.title);
-  });
-
-  it('can delete entire collection from system', () => {
-    cy.uploadCollection(collectionName, namespace.name, '1.0.0');
-    cy.getByDataCy('table-view').click();
-    cy.filterTableBySingleText(collectionName, true);
-    cy.clickLink(collectionName);
-    cy.verifyPageTitle(`${namespace.name}.${collectionName}`);
-    cy.contains('Loading').should('not.exist');
-    cy.selectDetailsPageKebabAction('delete-entire-collection-from-system');
-    // Verify collection has been deleted from system
-    cy.verifyPageTitle(Collections.title);
-    cy.getHubCollection(collectionName).then((deleted) => {
-      //Assert that the query returns an empty array, indicating no API results exist
-      expect(deleted.data).to.be.empty;
+  describe('Collections Details - all tests ', () => {
```

Comment:
naming TBD. We isolated the flaky tests to create and delete their resources within `beforeEach` and `afterEach` blocks.